### ISSUE TITLE
Update Module for V11

### ISSUE THREAD

**Issue by nschoenwald** [cdverrett94/rounded-measured-templates#9]
The module seems to work fine, but it uses some deprecated data that will have the backwards compatibility removed in V12:
<img width="517" alt="image" src="https://github.com/cdverrett94/FVTT-rounded-distance-for-measured-templates/assets/72390280/77a6f8e0-865e-4cc0-8402-808fa886f252">

**Comment by rmasoni** [comment]
Yes, please! This is one of those modules that is so good it should've been Core from the beginning.
